reservation/config/firebase-config.ts

```diff
@@ -1,18 +1,25 @@
 import { initializeApp } from "firebase/app";
 import { getAuth } from "firebase/auth";
 import { getFirestore } from "firebase/firestore";
-import Config from 'react-native-config';
+// import { 
+//   FIREBASE_API_KEY, 
+//   FIREBASE_AUTH_DOMAIN,
+//   FIREBASE_PROJECT_ID,
+//   FIREBASE_STORAGE_BUCKET,
+//   FIREBASE_MESSAGING_SENDER_ID,
+//   FIREBASE_APP_ID
+// } from '@env';
 
 const firebaseConfig = {
-  apiKey: Config.FIREBASE_API_KEY,
-  authDomain: Config.FIREBASE_AUTH_DOMAIN,
-  projectId: Config.FIREBASE_PROJECT_ID,
-  storageBucket: Config.FIREBASE_STORAGE_BUCKET,
-  messagingSenderId: Config.FIREBASE_MESSAGING_SENDER_ID,
-  appId: Config.FIREBASE_APP_ID,
-  measurementId: Config.FIREBASE_MEASUREMENT_ID
+  apiKey: 'AIzaSyBHDp1g65MfWi_tV5bvfZOdvTrqDu0tvtE',
+  authDomain: 'fly-reservation.firebaseapp.com',
+  projectId: 'fly-reservation',
+  storageBucket: 'fly-reservation.appspot.com',
+  messagingSenderId: '28137103397',
+  appId: '1:28137103397:web:c44488a03f1a5fb3a8dfcc'
```

Comment:
Use environment variables in your application, it is a bad practice to leave your keys exposed.